paper.md

```diff
@@ -0,0 +1,64 @@
+---
+title: 'Qiskit Dynamics: A Python package for simulating the time dynamics of quantum systems'
+tags:
+  - Python
+  - quantum
+  - quantum computer
+  - pulse
+  - control
+authors:
+  - name: Daniel Puzzuoli
+    corresponding: true
+    affiliation: 1
+  - name: Christopher J. Wood
+    affiliation: 2
+  - name: Daniel J. Egger
+    affiliation: 3
+  - name: Benjamin Rosand
+    affiliation: 4
+  - name: Kento Ueda
+    affiliation: 5
+affiliations:
+ - name: IBM Quantum, IBM Canada, Vancouver, BC, Canada
+   index: 1
+ - name: IBM Quantum, IBM T.J. Watson Research Center, Yorktown Heights, NY, USA
+   index: 2
+ - name: IBM Quantum, IBM Research Europe - Zurich, Ruschlikon, Switzerland
+   index: 3
+ - name: Yale University, New Haven, CT, USA
+   index: 4
+ - name: IBM Quantum, IBM Research Tokyo, Tokyo, Japan
+   index: 5
+
+
+date: 8 September 2023
+bibliography: paper.bib
+---
+
+# Summary
+
+Qiskit Dynamics is an open source Python library for numerically simulating the time dynamics of finite-dimensional quantum systems. The goal of the package is to provide flexible configuration of the numerical methods used for simulation: general tools for transforming models of quantum systems for more efficient simulation (rotating frames and the rotating wave approximation), choice of array representations (dense v.s. sparse, and different array libraries), and access to different types of underlying solvers (standard ODE v.s. geometric solvers). The package also contains advanced functionality for computing time-dependent perturbation theory expressions used in robust quantum control optimization `[@perturb1; @perturb2]`. 
+
+As part of the Qiskit Ecosystem (https://qiskit.org/ecosystem), the package interfaces with other parts of Qiskit `[@Qiskit]`. Most notably, Qiskit Dynamics provides tools for simulating control sequences specified by Qiskit Pulse `[@alexander_qiskit_2020]` which gives a pulse representation of quantum circuit instructions. Higher level interfaces allow users to build and interact with simulation-based objects that target the same constraints (coupling map, timing, etc.) as a specified IBM Quantum computer.
```

Comment:
What does `timing` indicate?
I find that the name `coupling map` may not be familiar to general users. How about deleting it or adding an explanation?

Reply:
Timing indicates the `dt` for the backend. I think I'd still like to keep this to elaborate on the what "constraints" are, but maybe some other more colloquial words would be better. E.g. "qubit layout" instead of "coupling map", and "sampling rate" instead of "timing".

Reply:
@to24toro I've just edited this, let me know what you think.

Reply:
Thanks, @DanPuzzuoli .

LGTM!